packages/dx-react-scheduler/src/plugins/drag-drop-provider.jsx

```diff
@@ -76,7 +76,9 @@ export class DragDropProvider extends React.PureComponent {
 
   applyChanges(startTime, endTime, payload, startEditAppointment, changeAppointment) {
     startEditAppointment({ appointmentId: payload.id });
-    changeAppointment({ change: { startDate: startTime, endDate: endTime } });
+    changeAppointment({
+      change: { startDate: startTime, endDate: endTime, childId: payload.childId },
```

Comment:
Maybe rename this to something like occurenceId?

Reply:
Now, I am not sure this variable is required. I remove this variable. We will implement recurrence editing in the future and will add this variable as you suggest `occurenceId`.